apps/sms/style/attachment.css

```diff
@@ -164,6 +185,15 @@
   background-position: left -30rem;
 }
 
+.ref-placeholder {
```

Comment:
Please don't override the ref type with vcrad icon here. As I said before vcard is not the only one that could classified as ref, vcalander is another possible format.